proxy/src/main/java/com/wavefront/agent/PushAgent.java

```diff
@@ -94,15 +96,6 @@
 import com.yammer.metrics.Metrics;
 import com.yammer.metrics.core.Counter;
 
-import io.grpc.netty.NettyServerBuilder;
-import io.netty.channel.ChannelHandler;
-import io.netty.channel.ChannelOption;
-import io.netty.handler.codec.LengthFieldBasedFrameDecoder;
-import io.netty.handler.codec.bytes.ByteArrayDecoder;
-import io.netty.handler.codec.http.HttpMethod;
-import io.netty.handler.codec.http.cors.CorsConfig;
-import io.netty.handler.codec.http.cors.CorsConfigBuilder;
-import io.netty.handler.ssl.SslContext;
```

Comment:
Can you revert this "format changes" to keep the diff smaller ?

Reply:
Reverted.